open_samus_returns_rando/files/templates/custom_init.lua

```diff
@@ -1,8 +1,10 @@
 Game.ImportLibrary("system/scripts/init_original.lua")
 
-Init.tNewGameInventory = {
-    TEMPLATE("new_game_inventory")
-}
+Init.tNewGameInventory = TEMPLATE("new_game_inventory")
+
+Init.fEnergyPerTank = TEMPLATE("energy_per_tank")
+Init.fAeionPerTank = TEMPLATE("aeion_per_tank")
```

Comment:
Now the values are written to this file but `Init.fAeionPerTank` itself is unused or is it used somewhere in the game internals?

Reply:
Do I even need them?